api-docs/events.md

```diff
@@ -91,7 +119,9 @@ POST /api/events
   "eventName": "Job Fair",
   "eventDate": "2019-09-24",
   "eventLocation": "Salt Lake City Library",
-  "totalAttendence": 150
+  "totalAttendence": 150,
+  "createdBy": 1,
```

Comment:
`createdBy` and `lastUpdatedBy` never need to be included in the request body because they are already included in the session cookie. We should remove them from POST and PATCH request body.